backend/app.py

```
@@ -152,7 +154,8 @@ def add_item():
         return jsonify({"message": "Item added successfully", "item": item})
 
     except Exception as e:
-        return jsonify({"error": str(e)}), 500
+        app.logger.error(f"Error adding item: {e}")
```

Ruff (0.8.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)